custom_components/givenergy_local/number.py

```
@@ -113,14 +113,14 @@ class GivEnergyNumberEntityDescription(NumberEntityDescription):
 # limits above (HR111/112) and are only meaningful on models that expose the
 # AC-config register block (HR300+). Gated via PlantCapabilities.has_ac_config_block
 # (and not is_three_phase — three-phase AC remaps the read-back to different registers
-# than the command writes; see modbus#75). The setter rejects values below 1, hence the
-# 1–100 range.
+# than the command writes; see modbus#75). Range is 0–100% to match the GivEnergy
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)